mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/UserReindexingTask.java

```diff
@@ -72,6 +91,10 @@ public String type() {
         return USER_RE_INDEXING;
     }
 
+    public User getUser() {
```

Comment:
I don't like that we leak task internal representation that way. Why do we need this public method? Are we able to find another solution?

Reply:
I will create the dto as a private class of the Task as Gauthier has done.